### PR TITLE
Feature: interactive list

### PR DESCRIPTION
**Pull request by memelopez**
this PR implements: 
- adds new JS file containing a method for checkboxes status change
- adds event listener for checkbox change
- adds Store class to deal with local storage operations
- add event listener and corresponding methods to add new task

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @memelopez 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 1]
```diff
@@ -1,18 +1,59 @@
...
+  // Get form values
+  const taskDescription = document.querySelector('#taskDesc').value.trim();
+  if (!validateDescription(taskDescription)) {
+    console.log('Error - empty description');
+  } else {
+    // Add task
+    UI.addTaskStore(taskDescription);
+
+    // reloads page
+    // eslint-disable-next-line no-restricted-globals
```
Comment (mikenath223) [inline src/index.js]: - [x] Please remove this eslint disable line it's best to fix the issue in the code.

File: src/index.js [hunk 2]
```diff
@@ -1,18 +1,59 @@
...
+
+    const ulList = document.querySelector('#task-list');
+    const itemChecked = e.target.parentElement.parentElement;
+    const nodes = Array.from(ulList.children);
+    const index = nodes.indexOf(itemChecked);
+
+    UI.taskCompleted(index, checkboxState);
+
+    // Reload page
+    // eslint-disable-next-line no-restricted-globals
```
Comment (mikenath223) [inline src/index.js]: - [x] Remove this eslint disable line also.

File: src/ui.js [hunk 3]
```diff
@@ -156,4 +160,19 @@ export default class UI {
     btmDiv.appendChild(pBtm);
     appDiv.appendChild(btmDiv);
   }
+
+  static addTaskStore(description) {
```
Comment (mikenath223) [inline src/ui.js:163]: - [x] Also remove the commented code on line 143 on this file.

**Review by carlos-ssh — COMMENTED** [review]
Hi @memelopez , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**